default/python/AI/AIstate.py

```diff
@@ -74,7 +74,12 @@ def convert_to_version(state, version):
 
     if version == 0:
```

Comment:
So this if-elif is just bad example.  Please add a docstring how to manage updates.